src/bin.ts

```diff
@@ -72,7 +72,7 @@ if (!args._.length || help) {
 }
 
 if (excludePath) {
-  exclude.push(getExcludeFunction(excludePath, cwd));
+  exclude.push(getExcludeFunction(cwd, excludePath));
```

Comment:
This looks like dead code actually, it appears I moved this into the “get exclude” function already.

Reply:
Then I'll remove that if statement.